### ISSUE TITLE
Look at related projects

### ISSUE THREAD

**Issue by thesamovar** [thesamovar/notpaper#6]
- [ ] SuppleScroll and SuppleVault https://github.com/boscoh/supplescroll

**Comment by rorybyrne** [comment]
Not completely relevant, but [obsidian.md](https://obsidian.md/) is an Electron app which sits on top of a local folder of (markdown) files. We could take inspiration from it.

**Comment by rorybyrne** [comment]
Two more related projects:

[Substance.io](https://substance.io/)  
[eLife Lens](https://github.com/elifesciences/lens) (which was mentioned in the Twitter thread right?)

Looks like there is some overlap in the people working on both of those projects. Maybe they have the same technology under the hood as well?

**Comment by rorybyrne** [comment]
Another pre-print publishing platform: [Quieos](https://www.qeios.com/)

**Comment by thesamovar** [comment]
Elsevier Enhanced Reader is not bad actually. Not as sophisticated as what we're trying here, but already works across all Elsevier journals as long as you have a subscription.

**Comment by thesamovar** [comment]
Qeios doesn't seem to have any relevant features as far as I can tell? It's just a standard article?

**Comment by thesamovar** [comment]
And Substance is just a closed source subcription-model editor that isn't even complete? Again, not sure we can learn much there.

**Comment by thesamovar** [comment]
In principle I like the graph view of obsidian, but not sure how helpful it looks in practice. Seems rather messy. I suspect most users will just switch it off.

**Comment by thesamovar** [comment]
Graph views are really hard to get right in my opinion.

**Comment by rorybyrne** [comment]
Yeah Qeios looks like it's a regular paper, but it [allows you to paste in the contents of a Word doc](https://twitter.com/qeios/status/1354191374170644481?s=20) which means there is some sort of parsing going on behind the scenes.

According to the [eLife Lens README](https://github.com/elifesciences/lens#credits), the Substance team is helping with the technical execution. In the `article/` directory of that repository, [it says](https://github.com/elifesciences/lens/tree/develop/article#lens-article) "The Lens Article Format is an implementation the Substance Document Model dedicated to scientific content". So I don't really know what's going on, but it seems like Lens and Substance are closely related?

I agree about Obsidian, visualizing a full graph is impractical and feels like a gimmick. But navigating through a graph feels natural and intuitive, so I think there's a parallel between navigating the graph of notes you're writing, and navigating the graph of research papers you're exploring.
